plutus-core/plutus-core/stdlib/PlutusCore/StdLib/Data/Data.hs

```diff
@@ -37,43 +36,39 @@ dataTy = mkTyBuiltin @_ @Data ()
 -- >       (fList : list data -> r)
 -- >       (fI : integer -> r)
 -- >       (fB : bytestring -> r) ->
--- >           chooseData
--- >               d
+-- >           matchData
 -- >               {unit -> r}
--- >               (\(u : unit) -> uncurry {integer} {list data} {r} fConstr (unConstrB d))
--- >               (\(u : unit) -> fMap (unMapB d))
--- >               (\(u : unit) -> fList (unListB d))
--- >               (\(u : unit) -> fI (unIB d))
--- >               (\(u : unit) -> fB (unBB d))
--- >               unitval
-caseData :: TermLike term TyName Name DefaultUni DefaultFun => term ()
-caseData = runQuote $ do
+-- >               fConstr
+-- >               fMap
+-- >               fList
+-- >               fI
+-- >               fB
+-- >               d
+matchData :: TermLike term TyName Name DefaultUni DefaultFun => term ()
```

Comment:
What's the point of having this, and why is it not just the `CaseData` builtin?

Reply:
It calls `CaseData` now. Previously it was a test that we can encode `CaseData` using `ChooseData` and the other builtins. Now we simply keep the test after updating it to use the new builtin. It's not a very helpful definition anymore, but it doesn't hurt to keep it around and it's used, so I don't wanna replace `matchData` with `caseData` at the call site in `PlutusCore.Examples.Data.Data.ofoldrData`, just wasted time.